middleware/webhooks.js

```diff
@@ -7,9 +7,16 @@ module.exports = function createWithWebhook({ secret, shopStore }) {
     const topic = request.get('X-Shopify-Topic');
     const shopDomain = request.get('X-Shopify-Shop-Domain');
 
+    // Shopify escapes forward slashes 
+    // + replaces '&' with \u0026 when the HMAC is built
+    // so we need to do the same otherwise the validation will fail
+    let message = JSON.stringify(data)
+    message = message.split('/').join('\\/');
```

Comment:
This shouldn't be necessary – Shopify does indeed escape forward slashes in the JSON serialization, but there shouldn't be any reason to explicitly replace forward slashes in the provided serialization. In fact, if anything I would expect validation to fail with this change, as forward slashes would now be _double_ escaped.

Here's an example in Ruby:

```
require 'openssl'
require 'base64'

req_body = %q({"id":8267063683,"title":"Test \/ With \/ Slash","body_html":"Try to index the SAS capacitor, maybe it will reboot the bluetooth hard drive!","vendor":"Casper LLC","product_type":"expedite next-generation architectures","created_at":"2016-09-01T17:54:41-04:00","handle":"antarctic-aluminum-poster","updated_at":"2017-11-13T13:27:07-05:00","published_at":"2016-09-01T17:54:41-04:00","template_suffix":"","published_scope":"global","tags":"newtag","variants":[{"id":27166731523,"product_id":8267063683,"title":"Adrift Steel Keyboard 9ee4234027d35b2bd8fe703ce546542a","price":"1675.99","sku":"","position":1,"inventory_policy":"deny","compare_at_price":null,"fulfillment_service":"manual","inventory_management":"shopify","option1":"Adrift Steel Keyboard 9ee4234027d35b2bd8fe703ce546542a","option2":null,"option3":null,"created_at":"2016-09-01T17:54:42-04:00","updated_at":"2017-04-07T15:20:10-04:00","taxable":false,"barcode":"","grams":3000,"image_id":null,"inventory_quantity":421,"weight":3.0,"weight_unit":"kg","old_inventory_quantity":421,"requires_shipping":false},{"id":27166731587,"product_id":8267063683,"title":"Belated Wooden Sandwich b21198570c9fa04d7183052da4e89721","price":"875.99","sku":"","position":2,"inventory_policy":"deny","compare_at_price":null,"fulfillment_service":"manual","inventory_management":"shopify","option1":"Belated Wooden Sandwich b21198570c9fa04d7183052da4e89721","option2":null,"option3":null,"created_at":"2016-09-01T17:54:42-04:00","updated_at":"2016-09-01T17:54:42-04:00","taxable":true,"barcode":null,"grams":2000,"image_id":null,"inventory_quantity":765,"weight":2.0,"weight_unit":"kg","old_inventory_quantity":765,"requires_shipping":true}],"options":[{"id":9922145155,"product_id":8267063683,"name":"Color or something","position":1,"values":["Adrift Steel Keyboard 9ee4234027d35b2bd8fe703ce546542a","Belated Wooden Sandwich b21198570c9fa04d7183052da4e89721"]}],"images":[],"image":null})

SECRET = "8de8c3b421c593e6f13c88dd3ee5220904fc222202228deb48ac0b162deae4e3"
digest = OpenSSL::Digest.new('sha256')
calculated_hmac = Base64.encode64(OpenSSL::HMAC.digest(digest, SECRET, req_body)).strip

puts calculated_hmac
```

Returns the correct HMAC value, even though the serialization contains several forward slashes.

Unless there is something weird happening in the internals of Node that I'm not aware of, this change should not be included. I'll continue to test.